src/tooltip/tooltip.js

```diff
@@ -76,6 +78,8 @@ MaterialTooltip.prototype.handleMouseLeave_ = function(event) {
 
   event.stopPropagation();
   this.element_.classList.remove(this.CssClasses_.IS_ACTIVE);
+  window.removeEventListener('scroll', this.boundMouseLeaveHandler);
+  window.removeEventListener('touchmove', this.boundMouseLeaveHandler, false);
```

Comment:
Would you mind moving these to the `mdlDowngrade_()` like all the other event handlers?